src/main/java/mate/filter/AuthenticationFilter.java

```diff
@@ -0,0 +1,40 @@
+package mate.filter;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+public class AuthenticationFilter implements Filter {
+    private Set<String> allowed = new HashSet<>();
+
+    @Override
+    public void init(FilterConfig filterConfig) {
+        allowed.add("/login");
```

Comment:
Maybe you can add  allowed.add("/"), becouse we can't show index's page. But it's ok, in my opinion.